src/guide/essentials/reactivity-fundamentals.md

```diff
@@ -251,56 +370,31 @@ export default {
 
 </div>
 
-### Reactividad Profunda {#deep-reactivity}
+<div class="composition-api">
 
-En Vue, el estado es profundamente reactivo por defecto. Esto significa que puedes esperar que se detecten cambios incluso cuando mutas objetos o arrays anidados:
+## `reactive()` \** {#reactive}
 
-<div class="options-api">
+Existe otra forma de declarar estado reactivo, con la API `reactive()`. A diferencia de una ref, que envuelve el valor interno en un objeto especial, `reactive()` hace reactivo al objeto mismo:
 
 ```js
-export default {
-  data() {
-    return {
-      obj: {
-        nested: { count: 0 },
-        arr: ['foo', 'bar']
-      }
-    }
-  },
-  methods: {
-    mutateDeeply() {
-      // estos funcionarán como se esperaba.
-      this.obj.nested.count++
-      this.obj.arr.push('baz')
-    }
-  }
-}
-```
-
-</div>
+import { reactive } from 'vue'
 
-<div class="composition-api">
+const state = reactive({ count: 0 })
+```
 
-```js
-import { reactive } from 'vue'
+> Consulta también: [Escritura de `reactive()`](/guide/typescript/composition-api#typing-reactive) <sup class="vt-badge ts"/>
 
-const obj = reactive({
-  nested: { count: 0 },
-  arr: ['foo', 'bar']
-})
+Uso en plantillas:
 
-function mutateDeeply() {
-  // estos funcionarán como se esperaba.
-  obj.nested.count++
-  obj.arr.push('baz')
-}
+```vue-html
+<button @click="state.count++">
+  {{ state.count }}
+</button>
 ```
 
-</div>
-
-También es posible crear explícitamente [objetos reactivos superficiales](/api/reactivity-advanced#shallowreactive) donde la reactividad solo se rastrea en el nivel raíz; sin embargo, normalmente estos solo se necesitan en casos de uso avanzado.
+Los objetos reactivos son [proxies de JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) y se comportan como objetos normales. La diferencia es que Vue es capaz de interceptar el acceso y mutación de todas las propiedades de un objeto reactivo para seguir y gatillar la reactividad.
```

Comment:
how about "La diferencia es que Vue es capaz de interceptar el acceso y la mutación de todas las propiedades de un objeto reactivo para el seguimiento y activación de la reactividad."? still not sure on 'gatillar'